nansat/nansat.py

```diff
@@ -1585,7 +1585,7 @@ def _import_mappers(log_level=None):
         value: class Mapper(VRT) from the mappers module
 
     """
-    logger = add_logger('import_mappers', logLevel=log_level)
+    logger = add_logger('Nansat', logLevel=log_level)
```

Comment:
this function is not part of the `Nansat` class

Reply:
but it is in nansat repository.DO you have any other name in your mind?

Reply:
Either leave it as it was or use something like `nansat.import_mappers`.

Note: `nansat` refers to the package. `Nansat` refers to the class.

Reply:
Done